flow/platforms/asap7/config.mk

```diff
@@ -111,8 +111,12 @@ export SET_RC_TCL              = $(PLATFORM_DIR)/setRC.tcl
 
 # Route options
 export MIN_ROUTING_LAYER       = M2
+export MIN_CLOCK_ROUTING_LAYER = M4
 export MAX_ROUTING_LAYER       = M7
 
+# Define fastRoute tcl
+export FASTROUTE_TCL ?= $(PLATFORM_DIR)/fastroute.tcl
```

Comment:
I don't think this is necessary.  In global_route.tcl just check if MIN_CLOCK_ROUTING_LAYER  is defined and if so do

set_routing_layers -clock $env(MIN_CLOCK_ROUTING_LAYER)-$env(MAX_ROUTING_LAYER)